feed-rs/src/xml/mod.rs

```diff
@@ -290,7 +297,7 @@ impl<R: BufRead> SourceState<R> {
 
                 // CData is converted to text
                 Event::CData(t) => {
-                    let escaped = t.escape();
+                    let escaped = t.escape()?;
```

Comment:
Not sure whether this error should be bubbled up or use to return a default instead.

Reply:
Upon reading the surrounding code it seems like the pattern is to skip rather than stop and return an error so I've pushed another commit to implement that instead.

Reply:
I think that is reasonable, thanks ... data that cannot be unescaped is problematic, potentially even a security concern.